tests/integrations/vertexai/vertexai_test.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+
+from weave.integrations.integration_utilities import op_name_from_ref
+
+
+@pytest.mark.retry(max_attempts=5)
+@pytest.mark.skip_clickhouse_client
+def test_content_generation(client):
+    import vertexai
+    from vertexai.generative_models import GenerativeModel
+
+    vertexai.init(project="wandb-growth", location="us-central1")
+    model = GenerativeModel("gemini-1.5-flash")
+    model.generate_content("Explain how AI works in simple terms")
+
+    calls = list(client.calls())
+    assert len(calls) == 1
+
+    call = calls[0]
+    assert call.started_at < call.ended_at
+
+    trace_name = op_name_from_ref(call.op_name)
+    assert trace_name == "vertexai.GenerativeModel.generate_content"
+    assert call.output is not None
```

Comment:
can you make a more relevant assert using a mock?  This is just checking if anything populated at all.

Reply:
I added cassettes for `test_content_generation` and `test_content_generation_stream`. However, I'm unable to generate them for `test_content_generation_async` and `test_content_generation_async_stream`.

Reply:
Do cassettes work here?  I thought this used the google grpc stuff under the hood?

In the case where they don't work, you (or gpt!) will need to mock out the request/response manually

Reply:
@andrewtruong Do you mean that we need to write the cassettes manually for those specific functions? Or is there another way?